src/filters/util.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+angular.module('openshiftCommonUI')
+  .filter("toArray", function() {
+    return function (items) {
+      if (!items) {
+        return [];
+      }
+
+      if (angular.isArray(items)) {
+        return items;
+      }
+
+      var itemsArray = [];
+      angular.forEach(items, function (item) {
+        itemsArray.push(item);
+      });
+
+      return itemsArray;
+    };
+  })
+  .filter('hashSize', function() {
```

Comment:
This can be a follow on, but I think we should add a `_.size` filter.

```js
.filter('size', function() {
  return _.size;
});
```

This is better since it handles arrays, too. I think we should deprecate `hashSize`.